OrderApi/Migrations/OrdersContextModelSnapshot.cs

```diff
@@ -0,0 +1,83 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using ShoesOnContainers.Services.OrderApi.Data;
+
+namespace OrderApi.Migrations
+{
+    [DbContext(typeof(OrdersContext))]
+    partial class OrdersContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "2.1.1-rtm-30846")
+                .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+
+            modelBuilder.Entity("ShoesOnContainers.Services.OrderApi.Models.Order", b =>
+                {
+                    b.Property<int>("OrderId")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<string>("Address");
+
+                    b.Property<string>("BuyerId");
+
+                    b.Property<string>("FirstName");
+
+                    b.Property<string>("LastName");
+
+                    b.Property<DateTime>("OrderDate");
+
+                    b.Property<int>("OrderStatus");
+
+                    b.Property<decimal>("OrderTotal");
+
+                    b.Property<string>("PaymentAuthCode");
+
+                    b.Property<string>("UserName");
+
+                    b.HasKey("OrderId");
+
+                    b.ToTable("Orders");
+                });
+
+            modelBuilder.Entity("ShoesOnContainers.Services.OrderApi.Models.OrderItem", b =>
```

Comment:
I believe these items will be automatically changed when migration is run AFTER you change the property and namespace names. Double check this file after you create your own migration.